src/pages/guide/comparison-with-leading-frameworks.ejs

```diff
@@ -30,12 +30,12 @@ const data = {
       <tr>
         <th></th>
         <th>
-          <img src="https://expr.nightwatchjs.org/images/symbol-nightwatch.svg" class="nightwatch-logo" />
+          <img src="https://expr.nightwatchjs.org/images/symbol-nightwatch.svg" class="nightwatch-logo" alt="nightwatch-logo" />
```

Comment:
The alt attribute should be a descriptive text field

Reply:
Noted. Sure!!